test/NeoSharp.VM.Test/Tests/PUSHM1_to_PUSH16.json

```diff
@@ -0,0 +1,211 @@
+﻿{
+    "category": "Simple PUSH",
+    "name": "From PUSHM1 to PUSH16 [-1 to 16]",
+    "tests":
+    [
+        {
+            "script": "TwBRUlNUVVZXWFlaW1xdXl9gZg==",
+            "trigger": "Application",
+            "steps":
+            [
+                {
+                    "actions":
+                    [
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto",
+                        "StepInto"
+                    ],
+                    "state":
+                    {
+                        "state": "None",
+                        "consumedGas": 0,
+                        "invocationStack":
+                        [
+                            {
+                                "scriptHash": "W2jhyKUs0WrgEZhOJr5tY35y7NU=",
+                                "instructionPointer": 18,
+                                "nextInstruction": 0x66,
+                                "evaluationStack":
+                                [
+                                    {
+                                        "type": "Integer",
```

Comment:
JSON is really verbose here. I think about micro syntax for the sake of clarity.
```
[
    "int 16",
    "int 15",
    "int 14",
    "int 13",
    "int 12",
    "int 11",
    "int 10",
    "int 9",
    "int 8",
    "int 7",
    "int 6",
    "int 5",
    "int 4",
    "int 3",
    "int 2",
    "int 1",
    "byte []",
    "int -1"
]
```

Reply:
I like it, could be easier to define like this

Reply:
Then scrap this idea. It starts looking as yaml file 😉